waspc/examples/todoApp/todoApp.wasp

```diff
@@ -50,11 +71,17 @@ query getTasks {
 }
 
 query getTask {
-  fn: import { getTask } from "@ext/queries.js"
+  fn: import { getTask } from "@ext/queries.js",
+  entities: [Task]
 }
 
 // --------- Actions --------- //
 
+action signUp {
+  fn: import { signUp } from "@ext/actions.js",
+  entities: [User]
```

Comment:
Hm, this kind of breaks our "injection" concept because even if you did not put entities: [User] here, signUp would still work. So you need to remember it and that is tricky. Maybe createNewUser should be also injected then once you add User?
I guess what would make most sense is if createNewUser was an action, which uses User, and then if your action uses that Action, we can figure out that it also depends on User. But that is somewhat advanced. So question is what do we do in meantime, do we go with smeth in between like I suggested and preserve this "injection" safety, or we keep it as it is until, with invalidation of queries broken until we have complete solution?